doc/whats_new.rst

```diff
@@ -20,20 +20,21 @@ NEW
   :func:`nilearn.plotting.plot_carpet` generates a "carpet plot" (also known
   as a "Power plot" or a "grayplot"), for visualizing global patterns in
   4D functional data over time.
+- The position annotation on the plot methods now implements the `decimals` option
+  to enable annotation of a slice coordinate position with the float.
 
 Fixes
 -----
 
 - :class:`nilearn.input_data.NiftiLabelsMasker` no longer ignores its `mask_img`
-- :func:`nilearn.masking.compute_brain_mask` has replaced
-  `nilearn.masking.compute_gray_matter_mask`. Features remained the same but
+- :func:`nilearn.masking.compute_brain_mask` has been replaced by
+  `:func: nilearn.masking.compute_gray_matter_mask`. Features remained the same but
   some corrections regarding its description were made in the docstring.
 
 
 Changes
 -------
 
-- `nilearn.datasets.fetch_surf_fsaverage5` has been removed.
```

Comment:
why is this removed? seems unrelated to this PR

Reply:
@jeromedockes This is not the change I made, its happened when I merged my fork with original